pages/common/docker.md

```diff
@@ -31,6 +31,10 @@
 
 `docker rm {{container_name}}`
 
+- Remove all stopped containers:
+
+`docker rm $(docker ps -aq)`
+
```

Comment:
Is there any less important example that we can think on removing?

Reply:
There are two examples about list containers

- `docker ps`
- `docker ps -a`

and if we take this example, there will be two `docker rm` example. Should we consider only take one here, because we can use the `docker ps --help` to show more options.

Reply:
This example is a composition of two previous examples. Primarily tldr pages only mention the basic skeletal examples upon which the user is free to build up more complex versions. Based on that, I don't think we need this.

Reply:
That is precisely why the example is important, because the idiomatic way of operating on multiple docker containers is by composing commands; and not by supplying some flag on the command line easily discovered through the man page.

Reply:
Sure, it may be important. But then content like that is suited for a different site like https://www.commandlinefu.com/ or similar cheat sheet sites. Our scope at tldr pages is to remain short and simple, by just providing the basic commands. A man page equivalent with helpful examples.